features/backend/product_archetypes.feature

```diff
@@ -76,6 +76,7 @@ Feature: Product archetypes
         Then I should be on the product archetype index page
         And I should see "Archetype has been successfully created."
 
+    @javascript
     Scenario: Creating a product by building archetype
```

Comment:
It's a good question, but I think that if there is no separate scenario for creating product **without** attributes, it can be left as it is ;)